front/cypress/e2e/report_builder/idea_template.cy.ts

```diff
@@ -115,23 +115,21 @@ describe('Idea template', () => {
           `/admin/reporting/report-builder/${reportId}/editor?templateProjectId=${projectId}`
         );
 
-        cy.wait(1000);
+        cy.wait(2000);
 
         // Edit text
         cy.get('.e2e-text-box').eq(2).click('center');
         cy.get('.ql-editor').click();
         const text = randomString();
 
-        cy.wait(1000);
+        cy.wait(2000);
```

Comment:
Sometimes cypress crashes my browser (especially when testing content builders), and adding random `cy.wait` statements seems to work (this is why they were there in the first place). But apparently I was not waiting long enough